data-model/fdo-types/0.3.0/digital-specimens/examples/aves-example.json

```diff
@@ -0,0 +1,137 @@
+{
+  "ods:ID": "https://doi.org/SANDBOX/XY7-ZMP-5J4",
+  "ods:version": 1,
+  "ods:created": "2024-05-29T07:10:56.269509Z",
+  "ods:type": "https://doi.org/21.T11148/894b1e6cad57e921764e",
+  "ods:midsLevel": 1,
+  "ods:normalisedPhysicalSpecimenID": "https://data.biodiversitydata.nl/naturalis/specimen/RMNH.AVES.121559",
+  "ods:physicalSpecimenID": "https://data.biodiversitydata.nl/naturalis/specimen/RMNH.AVES.121559",
+  "ods:physicalSpecimenIDType": "Resolvable",
+  "ods:topicOrigin": "Natural",
+  "ods:topicDomain": "Life",
+  "ods:topicDiscipline": "Zoology",
+  "ods:hasMedia": true,
+  "ods:specimenName": "Pitta versicolor Swainson, 1825",
+  "ods:sourceSystem": "https://hdl.handle.net/SANDBOX/CS1-ZKQ-NZ7",
+  "ods:livingOrPreserved": "Preserved",
+  "dcterms:license": "CC0 1.0",
+  "dcterms:modified": "2023/01/24",
+  "dwc:basisOfRecord": "PreservedSpecimen",
+  "dwc:preparations": "skin (air dried)",
+  "dwc:institutionID": "https://ror.org/0566bfb96",
+  "dwc:institutionName": "Naturalis Biodiversity Center",
+  "dwc:recordedBy": "van Lansberge",
+  "dwc:datasetName": "Naturalis Biodiversity Center (NL) - Aves",
+  "dcterms:rightsHolder": "Naturalis Biodiversity Center",
+  "ods:MaterialEntities": [],
+  "ods:Identifications": [
```

Comment:
Ok so instead of having the dwc:identification, etc. as an array we now have the ods variant as the array.
Guess this makes sense because now all of the dwc elements are objects

Reply:
This changed a bit to `ods:hasIdentification` : [{}]